aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1313,7 +1313,7 @@ std::vector<Tensor> unsafe_split(const Tensor& self, int64_t split_size, int64_t
   auto result = at::native::split(self, split_size, dim);
   for (auto& t : result) {
     // TODO(Ailing): do we need to set version_counter here?
-    if (!t.unsafeGetTensorImpl()->is_inference_tensor()) {
+    if (!t.unsafeGetTensorImpl()->is_inference()) {
```

Comment:
After the change there will be a `is_inference` method on `Tensor` as well, so I think it's safe to remove most `unsafeGetTensorImpl()` calls :D

Reply:
Good point!